### PR TITLE
Cat mode

### PR DESCRIPTION
**Pull request by toriagibbs-zz**
The much anticipated CAT MODE is here!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,5 +43,10 @@ You can override the following env variables:
 
 If you want to live **really** dangerously, why not combine Shark Attack with a healthy sprinkling of `shopt -s expand_aliases` in your production scripts? (*Warning:* this will almost certainly get you fired. [Hooray!](http://www.yellow5.com/pokey/archive/index86.html))
 
-## 4. License
+## 4. Cat Mode
+Terrified of sharks? Obsessed with cats? Live luxuriously with sharkattack: Cat Mode!
+* Follow the same configuration steps as above, but copy the very important `cats` folder into your data destination.
+* Use `cat_alias` instead of boring old `shark_alias` to add a little cat action to your life.g
```
Comment (cirla) [inline README.md]: Remove trailing `g` at the end.

File: sharkattack.sh [hunk 2]
```diff
@@ -19,6 +22,24 @@ function shark_alias {
     command=$2
     threshold=$3
 
+    SHARK_ATTACK_TYPE='shark'
```
Comment (cirla) [inline sharkattack.sh]: Setting these within the `*_alias` functions won't have any effect as `SHARK_ATTACK_TYPE` isn't de-referenced until `shark_swim` is actually called (or maybe it's dereferenced when `shark_swim` is defined, which would mean `shark_alias` and `cat_alias` would either both result in sharks or both result in cats). You probably want to scrap the `SHARK_ATTACK_TYPE` from this file and make a `cat_meow` function analogous to `shark_swim` to complement `cat_alias`.

**Comment by cirla** [comment]
:heart_eyes_cat:

**Comment by toriagibbs-zz** [comment]
:smile_cat:

**Pull request merged**